docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -528,6 +528,153 @@ Required ECS fields when not using {beats}:::
+be due to password spraying, user enumeration or brute force activity.
+
+Job details:::
+
+* Detects anomalies where the number of events is unusually high and ignores
+cases where the count is zero (using the
+{ml-docs}/ml-count-functions.html#ml-nonzero-count[`high_non_zero_count` function]).
+
+Required {beats} or {agent} integrations:::
+
+* {elastic-endpoint} integration
+
+Required ECS fields:::
+
+* `event.category`
+* `event.outcome`
+
+auth_high_count_logon_events_for_a_source_ip::
+Looks for an unusually large spike in successful authentication events events
+from a particular source IP address. This can be due to password spraying, user
+enumeration or brute force activity.
+
+Job details:::
+ 
+* Detects anomalies where the number of events by source IP is unusually high
+and ignores cases where the count is zero (using the
+{ml-docs}/ml-count-functions.html#ml-nonzero-count[`high_non_zero_count` function]).
+
+Required {beats} or {agent} integrations:::
+
+* {elastic-endpoint} integration
+
+Required ECS fields:::
+
+* `event.category`
+* `event.outcome`
+* `source.ip`
+* `user.name`
+* `winlog.event_data.LogonType`
+
+auth_high_count_logon_fails::
+Looks for an unusually large spike in authentication failure events. This can be
+due to password spraying, user enumeration or brute force activity and may be a
+precursor to account takeover or credentialed access.
+
+Job details:::
+ 
+* Detects anomalies where the number of events is unusually high and ignores
+cases where the count is zero (using the
+{ml-docs}/ml-count-functions.html#ml-nonzero-count[`high_non_zero_count` function]).
+
+Required {beats} or {agent} integrations:::
+
+* {elastic-endpoint} integration
+
+Required ECS fields:::
+
+* `event.category`
+* `event.outcome`
+
+auth_rare_hour_for_a_user::
+Looks for a user logging in at a time of day that is unusual for the user. This
+can be due to credentialed access via a compromised account when the user and
+the threat actor are in different time zones. In addition, unauthorized user
+activity often takes place during non-business hours.
+
+Job details:::
+ 
+* Detects anomalies where events happen at unusual times for a user (using the
+{ml-docs}/ml-time-functions.html#ml-time-of-day[`time_of_day` function]).
+
+Required {beats} or {agent} integrations:::
+
+* {elastic-endpoint} integration
+
+Required ECS fields:::
+
+* `event.category`
+* `event.outcome`
+* `source.ip`
+* `user.name`
+
+
+auth_rare_source_ip_for_a_user::
+Looks for a user logging in from an IP address that is unusual for the user.
+This can be due to credentialed access via a compromised account when the user
+and the threat actor are in different locations. An unusual source IP address
+for a username could also be due to lateral movement when a compromised account
+is used to pivot between hosts.
+
+Job details:::
+ 
+* Detects unusually rare `source.ip` values compared to other users (using the
```

Comment:
This doesn't sound right to me. Isn't it detecting rare `source.ip` values that are rare for that user (rather than comparing it to other users)? The description above looks like a better description `Looks for a user logging in from an IP address that is unusual for the user.`

Reply:
Thanks!  I've updated this bullet to match what we use in the job wizard (2a5182f0b6da14999925859f524576dbce610095)